spec/features/post/post_show_integraion_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+RSpec.configure do |config|
+  config.before(:each, type: :system) do
+    driven_by :selenium, using: :chrome, options: {
+      binary: '/usr/bin/google-chrome' # Replace with the actual Chrome binary location on your Chromebook
```

Comment:
- You have done well separating the tests into separate files as suggested by the previous reviewer, but there are a few changes you need to make before moving to the next milestone. :+1: :100: 

- - Error screenshot:
![Screenshot from 2023-11-16 09-42-03](https://github.com/Aqurds/coding-thought/assets/107632780/1b6d8573-6b74-4b73-92eb-8b836ba71275)

- [ ] When I tried setting up your project to run the tests, I Got this error when I tried to migrate your database. The error indicates that the 'comments' migration is trying to reference a 'posts' table that hasn't been created yet. :+1: :+1: 

In Rails, the order of migration files matters, especially when there are associations between tables. Since a comment belongs to a post, the `posts` table should be created before the `comments` table.

To resolve this, you can either:

Change the timestamp in the `comment` migration file `20231025085325_create_posts.rb` to a value earlier than the `posts` migration file.
or
Rewrite the migration files so that the 'posts' table comes before the 'comments' table in the migration order.
:smile: :+1:  I'll suggest yu rectify this because a user can't setup your project to run the test with these errors. :small_airplane: :smile:

Reply:
However, I update the timestamp of post table earlier then comments & like table in migration. I think that will be ok now. Thanks!